service/src/main/java/bio/terra/workspace/service/iam/SamService.java

```diff
@@ -828,7 +842,9 @@ public Boolean status() {
   private void addWsmResourceOwnerPolicy(CreateResourceRequestV2 request)
       throws InterruptedException {
     try {
-      String wsmSaEmail = getEmailFromToken(getWsmServiceAccountToken());
+      AuthenticatedUserRequest wsmRequest =
+          new AuthenticatedUserRequest().token(Optional.of(getWsmServiceAccountToken()));
```

Comment:
likewise (separately), we should change this signature not to take in an `Optional`, per the java guide.

Reply:
PF-1001